desktop/ELECTRON_ENVIRONMENT.js

```diff
@@ -1,30 +0,0 @@
-// This variable is injected into package.json by electron-builder via the extraMetadata field (specified in electronBuilder.ghactions.config.js)
-// It will be `PROD` on production, `STG` on staging, and `undefined` on dev (because dev doesn't use electron-builder)
-const {electronEnvironment} = require('../package.json');
-const ENVIRONMENT = require('../src/CONST/ENVIRONMENT');
```

Comment:
Since `src/CONST/ENVIRONMENT` isn't used by any CJS module anymore, can we please combine `src/CONST/ENVIRONMENT.js` and `src/CONST/index.js` back together into a single `src/CONST.js` file?

Reply:
Sure